admin_manual/maintenance/docker/index.rst

```diff
@@ -0,0 +1,96 @@
+``DOMAIN``         The ownCloud domain            ``localhost``
+``ADMIN_USERNAME`` The admin username             ``admin``
+``ADMIN_PASSWORD`` The admin user's password      ``admin``
+``HTTP_PORT``      The HTTP port to bind to       ``80``
+``HTTPS_PORT``     The HTTP port to bind to       ``443``
+================== ============================== =============
+
+Then, you can start the container, using your preferred Docker command-line tool. 
+The example below shows how to use `Docker Compose <https://docs.docker.com/compose/>`_. 
+
+.. note:: 
+   You can find instructions for using plain docker `in the GitHub repository <https://github.com/owncloud-docker/server#launch-with-plain-docker>`_.
+
+.. code-block:: console
+   
+   # Clone the repository
+   git clone https://github.com/owncloud-docker/server.git owncloud
+   
+   cd owncloud
+   
+   # Create the environment configuration file
+   cat << EOF > .env
+   VERSION=10.0.2
+   DOMAIN=localhost
+   ADMIN_USERNAME=admin
+   ADMIN_PASSWORD=admin
+   HTTP_PORT=80
+   HTTPS_PORT=443
+   EOF
+   
+   # Build and start the container
+   docker-compose up -d --build
+   
+When the process completes, then check that all the containers have successfully
+started, by running ``docker-compose ps``. 
+If they are all working correctly, you should expect to see output similar to
+that below:
+
+.. code-block:: console
+   
+            Name                     Command               State                     Ports                    
+   -------------------------------------------------------------------------------------------------------
+   server_db_1         /usr/bin/entrypoint /bin/s ...   Up      3306/tcp                                   
+   server_owncloud_1   /usr/local/bin/entrypoint  ...   Up      0.0.0.0:443->443/tcp, 0.0.0.0:8080->80/tcp 
+   server_redis_1      /bin/s6-svscan /etc/s6           Up      6379/tcp
+   
+In it, you can see that the database, ownCloud, and Redis containers are running, and that ownCloud is accessible via ports 443 and 8080 on the host machine.
+
+.. note::
+   Just because all the containers are running, it takes a few minutes for ownCloud to be fully functional. 
+   If you run ``docker-compose logs --follow owncloud`` and see a significant amount of information logging to the console, then please wait until it slows down to attempt to access the web UI.
+   
+Logging In
+^^^^^^^^^^
+
+To log in to the ownCloud UI, open ``https://localhost`` in your browser of
+choice, where you see the standard ownCloud login screen, as in the image
+below.
+
+.. image:: ../..//images/docker/owncloud-ui-login.png
+   :alt: The ownCloud UI via Docker
+   
+The username and password are the admin username and password which you stored in ``.env`` earlier.
+
+.. note:: 
+   The first time that you access the login page via HTTPS, a browser
+   warning appears, as the SSL certificate in the Docker setup is self-signed.
```

Comment:
Maybe mention that the certificate can be overwritten within the host volume with a valid cert?